tools/docker-compose-datasets-server.yml

```diff
@@ -240,6 +94,7 @@ services:
       PARQUET_AND_DATASET_INFO_SUPPORTED_DATASETS: ${PARQUET_AND_DATASET_INFO_SUPPORTED_DATASETS-}
       PARQUET_AND_DATASET_INFO_TARGET_REVISION: ${PARQUET_AND_DATASET_INFO_TARGET_REVISION-refs/convert/parquet}
       PARQUET_AND_DATASET_INFO_URL_TEMPLATE: ${PARQUET_AND_DATASET_INFO_URL_TEMPLATE-/datasets/%s/resolve/%s/%s}
+      WORKER_STATE_PATH: ${WORKER_STATE_PATH-/state/worker-state}
```

Comment:
The WORKER_STATE_PATH needs to be different every time the worker restarts. You shouldn't set a default value for it.

It's important otherwise we'll end up with executors with an outdated state

Reply:
oh, you're right!

Reply:
(even if in that case there is only one worker)

Reply:
updated